src/lib/Components/Bidding/Screens/BidResult.tsx

```diff
@@ -16,7 +18,7 @@ import { Title } from "../Components/Title"
 
 import { BidResult_sale_artwork } from "__generated__/BidResult_sale_artwork.graphql"
 
-const SHOW_TIMER_STATUSES = ["WINNING", "OUTBID", "RESERVE_NOT_MET"]
+const SHOW_TIMER_STATUSES = ["WINNING", "OUTBID", "RESERVE_NOT_MET", "ERROR_BID_LOW"]
```

Comment:
@sepans This was removed [here](https://github.com/artsy/emission/pull/1054/files#diff-daac583405fb039e935fb2fec6983390L19) but I'm not sure why. It seems like we want to show the bid-again button when `ERROR_BID_LOW` is the status, right? I think I'm missing something.

Reply:
I believe `ERROR_BID_LOW` was renamed to `OUTBID` (we still control this whole flow 😄)

Reply:
Yes we no longer need "ERROR_BID_LOW"